frontend/app/components/common/path-heleper/path-helper.service.js

```diff
@@ -31,60 +31,69 @@ angular
   .factory('PathHelper', PathHelper);
 
 function PathHelper() {
-  var PathHelper;
+  var PathHelper,
+      appBasePath = window.appBasePath ? window.appBasePath : '';
 
   return PathHelper = {
-    apiV2: '/api/v2',
-    apiExperimental: '/api/experimental',
-    apiV3: '/api/v3',
-
-    appBasePath:  window.appBasePath ? window.appBasePath : '',
     staticBase:   appBasePath,
```

Comment:
I'm not entirely sure, I think it's just for naming. But since `appBasePath` is simply a variable, I left the property name the same in order to avoid changes to outside references to `PathHelper.staticBase`.